packages/jest-runner/fuzz.ts

```diff
@@ -42,15 +42,19 @@ export class FuzzerStartError extends FuzzerError {}
 // Use Jests global object definition.
 const g = globalThis as unknown as Global.Global;
 
-export type FuzzTest = (name: Global.TestNameLike, fn: FuzzTarget) => void;
+export type FuzzTest = (
```

Comment:
Could you please extend the docs accordingly?

Reply:
Pending for now until we settle the meaning of the `-timeout` flag

Reply:
We could also additionally add a test in `jest_integration` to show the test based timeout functionality.

Reply:
Done